mesa/visualization/ModularVisualization.py

```diff
@@ -106,6 +107,11 @@
 
 from mesa.visualization.UserParam import UserSettableParameter
 
+template_path = os.path.join(
```

Comment:
Just for clarification: Why did you add this?

Reply:
As per commit message

"Obtain the template directory from the module installation path
not the specific file - allowing you to have variations of the file
in your model if required"